xc/xenops_server_xen.ml

```diff
@@ -97,6 +97,7 @@ module VmExtra = struct
 		qemu_vifs: (Vif.id * (int * qemu_frontend)) list;
 		pci_msitranslate: bool;
 		pci_power_mgmt: bool;
+		pv_drivers_detected: bool;
```

Comment:
When I look in the last_booted_record for a VM that I've suspended, I only see `last_start_time`, `build_info` and `ty`. These are the persistent fields but not the non-persistent fields.

Also the comment in the VmExtra module says:
>         (** Extra data we store per VM. The persistent data is preserved when
                the domain is suspended so it can be re-used in the following 'create'
                which is part of 'resume'. The non-persistent data will be regenerated.

Does that mean we're okay here?